google-factcheck-explorer-bot/google-factcheck-explorer-bot-lambda/index.js

```diff
@@ -0,0 +1,159 @@
+// it will call a search in Alegre to look for similar ClaimReview items in the 
+// GoogleFactCheck feed defined in the config. 
+exports.handler = (event, context, callback) => {
+    const data = JSON.parse(event.body);
+    console.log('JSON.parse(event.body)', data);
+    if (data.event === 'create_project_media') {
+      console.log('Google Fact Check bot processing project media creation event')
+      // there could be an error paylod instead, but we let it fail to surface
+      const pmid = data.data.dbid.toString();
+      const type = data.data.type;
+      //TODO: Decide whether to include imported reports ("Blank")
+      if(type=="Claim" || type=="Link" || type=="Blank"){ 
+        const title = data.data.title.toString();
+        const description = data.data.description.toString();
+        //TODO: if desription is different than the title, make a second request
+        const http = require('https');
+  
+        var options = {
+          hostname: CHECK_API_URL,
+          path: `/api/v2/feeds?filter\[query\]=${encodeURIComponent(title)}`,
+          headers: {
+              Accept: 'application/vnd.api+json',
+              'X-Check-Token': `${CHECK_API_GOOGLE_FACT_CHECK_ACCESS_TOKEN}`, // This API key has access to Check workspace with googleFactCheck items
+            },
+          method: 'GET'  // post not configurd
+        };
+
+        
+        var req = http.request(options, (res) => {
+          res.setEncoding('utf8');
+          let responseBody = '';
+        
+          res.on('data', (chunk) => {
+              responseBody += chunk;
+          });
+        
+          res.on('end', () => {
+            if (res.statusCode >= 400){
+              console.error('Request error status',res.statusCode)
+              console.log('Error response body:',responseBody);
+            }
+            console.log('response body:',responseBody);
+            const json_obj = JSON.parse(responseBody);
+            // check if anything matched {"data":[],"meta":{"record-count":0}}
+            if (json_obj["meta"]["record-count"] > 0){
+              // {"data":[{"id":"20007","type":"feeds","links":{"self":"https://qa-check-api.checkmedia.org/api/v2/feeds/20007"},
+              // "attributes":{"claim":"-","claim-context":null,"claim-tags":"",
+              // "fact-check-title":"Madhuri Dixit campaigning For Imran Khan?",
+              // "fact-check-summary":"Pakistan's PTI party is using Amitabh Bachchan and Madhuri Dixit photos on their campaign posters",
+              // "fact-check-published-on":1679572669,"fact-check-rating":"undetermined",
+              // "published-article-url":"https://www.indiatoday.in/fact-check/story/viral-test-big-b-madhuri-dixit-campaigning-for-imran-khan-1294131-2018-07-24",
+              // "organization":"Google fact check tools"}}],
+              // "meta":{"record-count":1}}
+              let text = `<h6>Closely releated ClaimReviews from Google Factcheck Tools:</h6>\n`;
```

Comment:
HTML does actually work? That's great.

Reply:
Nope, it didn't work, but I forgot to remove after testing! thank you
![Screenshot 2023-07-21 at 10 47 30 AM](https://github.com/meedan/check-bots/assets/122867176/f5c6b19f-0164-4da0-900f-f7d77221a5f5)